flows.json

```diff
@@ -4695,5 +4703,293 @@
+        ]
+    },
+    {
+        "id": "14e11cd7a7c79172",
+        "type": "exec",
+        "z": "a02e6d42663c576b",
+        "command": "curl -s -X GET --unix-socket /var/run/docker.sock http://localhost/containers/json?all=1",
+        "addpay": "",
+        "append": "",
+        "useSpawn": "false",
+        "timer": "",
+        "winHide": false,
+        "oldrc": false,
+        "name": "Fetch all containers",
+        "x": 410,
+        "y": 80,
+        "wires": [
+            [
+                "94217953a8099a53"
+            ],
+            [],
+            []
+        ]
+    },
+    {
+        "id": "94217953a8099a53",
+        "type": "switch",
+        "z": "a02e6d42663c576b",
+        "name": "",
+        "property": "payload",
+        "propertyType": "msg",
+        "rules": [
+            {
+                "t": "empty"
+            },
+            {
+                "t": "else"
+            }
+        ],
+        "checkall": "true",
+        "repair": false,
+        "outputs": 2,
+        "x": 590,
+        "y": 80,
+        "wires": [
+            [
+                "08d0cbd04d6f2367"
+            ],
+            [
+                "37fc88db9034e16c"
+            ]
+        ]
+    },
+    {
+        "id": "37fc88db9034e16c",
+        "type": "json",
+        "z": "a02e6d42663c576b",
+        "name": "",
+        "property": "payload",
+        "action": "obj",
+        "pretty": false,
+        "x": 910,
+        "y": 120,
+        "wires": [
+            [
+                "68821b8a2c7472b8"
+            ]
+        ]
+    },
+    {
+        "id": "b6a8459814dcfd36",
+        "type": "template",
+        "z": "a02e6d42663c576b",
+        "name": "Parse Containers Output and Transform to Prometheus",
+        "field": "payload",
+        "fieldType": "msg",
+        "format": "handlebars",
+        "syntax": "mustache",
+        "template": "# HELP netbox_docker_agent_container_running Show if a container is running\n# TYPE netbox_docker_agent_container_running gauge\n{{#container.running}}\nnetbox_docker_agent_container_running{name=\"{{Names.0}}\", state=\"{{ State }}\", status=\"{{ Status }}\"} 1\n{{/container.running}}\n\n# HELP netbox_docker_agent_container_exited Show if a container is exited\n# TYPE netbox_docker_agent_container_exited gauge\n{{#container.exited}}\nnetbox_docker_agent_container_exited{name=\"{{Names.0}}\", state=\"{{ State }}\", status=\"{{ Status }}\"} 1\n{{/container.exited}}\n\n# HELP netbox_docker_agent_container_stopped Show if a container is exited\n# TYPE netbox_docker_agent_container_stopped gauge\n{{#container.stopped}}\nnetbox_docker_agent_container_stopped{name=\"{{Names.0}}\", state=\"{{ State }}\", status=\"{{ Status }}\"} 1\n{{/container.stopped}}\n\n# HELP netbox_docker_agent_docker_daemon Show if the connection to the daemon is working\n# TYPE netbox_docker_agent_docker_daemon gauge\nnetbox_docker_agent_docker_daemon{socket=\"/var/run/docker.socket\"} {{container.docker_daemon}}",
```

Comment:
You need to add triple brace in order to remove HTML escaping by mustache (eg: `{{{ Status }}}`)

Reply:
it's fixed!